### PR TITLE
Add dropdown for previous logs

### PR DESCRIPTION
**Pull request by zlayne**
Signed-off-by: Zack Layne <zlayne@redhat.com>

Related issue: https://issues.redhat.com/browse/ACM-1967

### CODE REVIEW

File: frontend/src/routes/Home/Search/Details/LogsPage.tsx [hunk 1]
```diff
@@ -104,21 +121,58 @@ export function LogsToolbar(props: {
         return { downloadUrl, downloadFilename: downloadFile.name }
     }, [name, cluster, container, logs])
 
+    const previousLogsDropdown = useMemo(() => {
+        console.log('rendering previousLogsDropdown')
```
Comment (KevinFCormier) [inline frontend/src/routes/Home/Search/Details/LogsPage.tsx:125]: Meant to remove?

**Review by KevinFCormier — COMMENTED** [review]
Looks good. Just a console.log I think you probably meant to remove.

File: frontend/src/routes/Home/Search/Details/LogsPage.tsx (hunk 1 above)
Comment (zlayne) [reply frontend/src/routes/Home/Search/Details/LogsPage.tsx]: Just updated, thanks!

**Pull request merged**